libs/features/personalization/personalization.js

```diff
@@ -50,14 +50,21 @@ const DATA_TYPE = {
 };
 
 const createFrag = (el, url, manifestId) => {
-  const a = createTag('a', { href: url }, url);
+  let href = url;
+  try {
+    const { pathname, search, hash } = new URL(url);
```

Comment:
`normalizePath` in the manifest would drop the search and hash in the url.

Reply:
Fragment links don't use hash & search, so we should use normalizePath